public/apps/account/account-nav-button.tsx

```diff
@@ -76,14 +79,21 @@ export function AccountNavButton(props: {
         View roles
       </EuiButtonEmpty>
       {horizontalRule}
-      {
-        // TODO: show switch tenants modal
-      }
-      <EuiButtonEmpty size="xs">Switch tenants</EuiButtonEmpty>
+      <EuiButtonEmpty
+        size="xs"
+        onClick={() =>
+          setModal(
+            <TenantSwitchPanel
+              {...props}
+              username={props.username}
```

Comment:
I think the username is not needed as a property, the account info fetched in the component also gives you username.

Reply:
Sure. Removed.